src/main/java/com/ibm/cloud/sdk/core/util/DateDeserializer.java

```diff
@@ -113,8 +109,6 @@ public synchronized Date deserialize(JsonElement json, Type typeOfT, JsonDeseria
       return new Date(timeAsLong);
     }
 
-    LOG.log(Level.SEVERE, "Error parsing: " + dateAsString, e);
```

Comment:
While working on this issue, I discovered that we were just absorbing an exception caused by an incorrectly-formatted date/datetime value, so made this change to throw an exception in that case.